xml/cap_admin_app_autoscaler.xml

```diff
@@ -36,27 +36,18 @@
    </listitem>
    <listitem>
     <para>
-     <link xlink:href="https://docs.cloudfoundry.org/cf-cli/">The &cf; command
-     line interface (&cfcli;)</link>
+     The Cloud Foundry command line interface (cf CLI), see <link xlink:href="https://docs.cloudfoundry.org/cf-cli/"/>
     </para>
    </listitem>
    <listitem>
     <para>
-     <link xlink:href="https://github.com/cloudfoundry-incubator/app-autoscaler-cli-plugin">The
-     App-AutoScaler &cfcli; Plugin</link>
+     The Cloud Foundry CLI AutoScaler Plug-in <link      xlink:href="https://github.com/cloudfoundry-incubator/app-autoscaler-cli-plugin"/>
```

Comment:
`...AutoScaler Plug-in, see <link>`

Github still shows 5 spaces between `link` and `xlink` .e.g `<link     xlink:href="..."` and the same when I view it locally too. Perhaps it's a setting in your editor? This doesn't have any affect on the output though, so it's not a big deal.

Reply:
Ok, I found the place in my editor and fixed it. Yes, it seemed there were 5 spaces there.